### DISCUSSION TITLE
Unused port giving substantial readings

### DISCUSSION THREAD

**Discussion by catatung** [emporia-vue-local/esphome#235] [Q&A]
Hello, I've got this all set up and working correctly it seems to me, however I am only using ports 1-15.  15 is a double pole 30A breaker and so I am using the multiplier:2 on the configuration.  Port 16 is left unused and not connected to a CT sensor.  I've put the rubber plug inside the jack.

What I am seeing are readings jumping around between 70w-120w.  I don't understand why, however since I am not using this port it's not of much concern to me -- Should I plug in a CT sensor to try and zero this out, or should I disable this entity?  Or is it irrelevant what I do here, as total consumption is calculated based on the two main CT sensors on phase a/phase b?

Thanks for any suggestions
Randy

**Comment by flaviut** [comment]
Closing in order to consolidate this question all in one thread: https://github.com/emporia-vue-local/esphome/discussions/103

Please let me know if that doesn't help!
